changelog/22583.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+core/quotas: Reduce overhead for role calculation when using cloud auth methods.
```

Comment:
This is total bike-shedding, but wouldn't it be more accurate to say "Reduce overhead due to repeated role calculations when using cloud auth methods.